lib/distribute.js

```diff
@@ -176,6 +225,69 @@ function distributeTwoNodes(firstNode, secondNode) {
   return NodeCreator.parenthesis(NodeCreator.operator('+', newArgs));
 }
 
+// Simplifies a sum of terms (a result of distribution) that's in parens
+// (note that all results of distribution are in parens)
+// e.g. 2x*(4 + x) distributes to (2x*4 + 2x*x)
+// This is a separate function from simplify to make the flow more readable,
+// but this is literally just a wrapper around 'simplify'.
+// Returns a NodeStatus object
+function simplifyWithParens(node) {
```

Comment:
Hmm, I'm not sure that having this as a separate function actually makes this flow more readable